.github/workflows/contracts.yml

```diff
@@ -0,0 +1,72 @@
+name: Solidity
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "solidity/contracts/**"
+      - "solidity/package.json"
+      - "solidity/package-lock.json"
```

Comment:
Heh, we accidentally self-identified an issue here: these paths should include this file as well, otherwise the job won't run on changes to the job.

Reply:
right, added this file for existing events